Library/include/CSP/Systems/Users/Authentication.h

```diff
@@ -107,7 +110,7 @@ class CSP_API LoginStateResult : public csp::systems::ResultBase
 	/** @endcond */
 
 public:
-	LoginState& GetLoginState();
```

Comment:
Non-const getters for result types were removed due to them being unnecessary. Result types are meant to be immutable, at least where clients are concerned.